shell/browser/api/atom_api_web_contents.cc

```diff
@@ -1737,6 +1739,27 @@ bool WebContents::IsCurrentlyAudible() {
 }
 
 #if BUILDFLAG(ENABLE_PRINTING)
+void WebContents::OnGetDefaultPrinter(
+    base::Value print_settings,
+    printing::CompletionCallback print_callback,
+    base::string16 device_name,
+    bool silent,
+    base::string16 default_printer) {
+  base::string16 printer_name =
+      device_name.empty() ? default_printer : device_name;
+  print_settings.SetStringKey(printing::kSettingDeviceName, printer_name);
+
+  auto* print_view_manager =
+      printing::PrintViewManagerBasic::FromWebContents(web_contents());
```

Comment:
Continuing discussion about https://github.com/electron/electron/pull/21986#issuecomment-580972199 , the crash is caused by this line.

This function is registered as a weak callback to the task runner below, so that its not invoked when the `api::WebContents` is not available. Based on the crash it seems `content::WebContents` is not available in the callback, @zcbenz is it possible for `api::WebContents` to be alive while the `content::WebContents` is not ?